components/constellation/pipeline.rs

```diff
@@ -624,8 +609,10 @@ impl UnprivilegedPipelineContent {
         );
 
         if wait_for_completion {
-            let _ = self.script_content_process_shutdown_port.recv();
-            let _ = self.layout_content_process_shutdown_port.recv();
+            let signal = content_process_shutdown_port.recv();
+            // When the script-thread drops the sender,
+            // we expect to receive an Err(_).
+            assert!(signal.is_err());
```

Comment:
Perhaps an `error!` log rather than a panic?

Reply:
Oh yeah, forgot this is in the constellation thread...

Reply:
It's not in the constellation thread actually, but I guess an error is better.